electron-app/src/components/AccountManagement.vue

```diff
@@ -117,8 +120,8 @@ export default class AccountManagement extends Vue {
     const { username, password, apiKey, apiSecret } = credentials;
     this.loading = true;
     await this.$store.dispatch('session/unlock', {
-      username: username,
-      password: password,
+      username,
```

Comment:
what do you achieve with this change?

Reply:
You don't repeat the same thing twice. Basically if you do { username } then you have an object that has a property username with the value of the variable username.

Reply:
hmm interesting. I did not know of this syntax